Lib/enum.py

```diff
@@ -946,3 +946,8 @@ def _decompose(flag, value):
         # we have the breakdown, don't need the value member itself
         members.pop(0)
     return members, not_covered
+
```

Comment:
PEP 8 states that there should be two blank spaces here, but the rest of the file doesn't follow that rule.